perllib/FixMyStreet/Cobrand/Zurich.pm

```diff
@@ -944,86 +917,81 @@ sub stash_states {
             unconfirmed => 1,
             hidden => 1,
         },
         {
             # Aufgenommen
             state => 'confirmed',
-            trans => _('Open'),
             unconfirmed => 1,
         },
         {
             # Unsichtbar (hidden)
             state => 'hidden',
-            trans => _('Hidden'),
             unconfirmed => 1,
             hidden => 1,
         },
         {
             # Extern
-            state => 'closed',
-            trans => _('Extern'),
+            state => 'external',
         },
         {
             # Zustaendigkeit unbekannt
-            state => 'unable to fix',
-            trans => _('Jurisdiction unknown'),
+            state => 'jurisdiction unknown',
         },
         {
-            # Wunsch (hidden)
-            state => 'investigating',
-            trans => _('Wish'),
+            # Wunsch
+            state => 'wish',
         },
         {
             # Nicht kontaktierbar (hidden)
-            state => 'partial',
-            trans => _('Not contactable'),
+            state => 'not contactable',
         },
     );
-    my %state_trans = map { $_->{state} => $_->{trans} } @states;
 
     my $state = $problem->state;
 
     # Rueckmeldung ausstehend may also indicate the status it's working towards.
     push @states, do {
-        if ($state eq 'planned' and my $closure_status = $problem->get_extra_metadata('closure_status')) {
+        if ($state eq 'feedback pending' and my $closure_status = $problem->get_extra_metadata('closure_status')) {
             {
                 state => $closure_status,
-                trans => sprintf '%s (%s)', _('Planned'), $state_trans{$closure_status},
+                trans => sprintf 'Rückmeldung ausstehend (%s)', FixMyStreet::DB->resultset("State")->display($closure_status),
             };
         }
         else {
             {
-                state => 'planned',
-                trans => _('Planned'),
+                state => 'feedback pending',
             };
         }
     };
 
     if ($state eq 'in progress') {
         push @states, {
             state => 'in progress',
-            trans => _('In progress'),
         };
     }
     elsif ($state eq 'fixed - council') {
         push @states, {
             state => 'fixed - council',
-            trans => _('Closed'),
         };
     }
     elsif ($state =~/^(hidden|unconfirmed)$/) {
         @states = grep { $_->{$state} } @states;
     }
     $c->stash->{states} = \@states;
-    $c->stash->{states_trans} = { map { $_->{state} => $_->{trans} } @states }; # [% states_trans.${problem.state} %]
 
     # stash details about the public response
     $c->stash->{default_public_response} = "\nFreundliche Grüsse\n\nIhre Stadt Zürich\n";
     $c->stash->{show_publish_response} = 
-        ($problem->state eq 'planned');
+        ($problem->state eq 'feedback pending');
+}
+
+# Same as prettify_state, but with override for 'unconfirmed'
+sub zurich_prettify_state {
```

Comment:
Why not just override `prettify_state` itself? Are there places it's used where you don't want `unconfirmed` to say `Submitted`?